src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -1253,6 +1273,34 @@ private ICollectionExpressionOperation CreateBoundCollectionExpression(BoundColl
                         throw ExceptionUtilities.UnexpectedValue(expr.CollectionTypeKind);
                 }
             }
+
+            static ImmutableArray<IArgumentOperation> getCreationArguments(
+                CSharpOperationFactory @this, BoundCollectionExpression expr)
+            {
+                var collectionCreation = expr.CollectionCreation;
+                if (collectionCreation is { HasAnyErrors: true })
```

Comment:
Why is this the right thing to do? I would usually expect that in erroneous code, the public APIs still provide the information we do know about the code.

Reply:
@CyrusNajmabadi, what kinds of conversions could appear here?
@333fred, do those conversions need to be exposed in IOperation tree? (currently this drops/skips nodes)